core/src/main/resources/org/pentaho/platform/engine/services/runtime/plugins.properties

```diff
@@ -4,6 +4,7 @@
 # a deprecation warning to the Pentaho console.
 #
 # Map the Short Names
+MondrianFlushCacheComponent = org.pentaho.platform.plugin.action.mondrian.MondrianFlushCacheComponent
```

Comment:
Because of PPP-3817, the plugins.properties file is also in the WEB-INF/classes folder here: https://github.com/pentaho/pentaho-platform/blob/master/assemblies/pentaho-war/src/main/webapp/WEB-INF/classes/org/pentaho/platform/engine/services/runtime/plugins.properties 

You'll need to update that file in this PR as well.